### DISCUSSION TITLE
Does StereoKit have a way to generate vertices?

### DISCUSSION THREAD

**Discussion by TheEmeraldFalcon** [StereoKit/StereoKit#98] [General]
I want to make a VR game that has randomly generated worlds via 3D perlin noise. I wanted to know if there were any features that would allow me to create these vertices?

**Comment by maluoi** [comment]
Yes! It absolutely does :) Check out the docs for Mesh.SetVerts, which includes some sample code for generating a mesh.

https://stereokit.net/Pages/Reference/Mesh/SetVerts.html

**Reply by TheEmeraldFalcon** [reply]
Thank you!
